pkg/kapp/clusterapply/applying_changes.go

```diff
@@ -95,7 +95,7 @@ func (c *ApplyingChanges) Apply(allChanges []*ctldgraph.Change) ([]WaitingChange
 			}
 
 			c.markApplied(result.Change)
-			appliedChanges = append(appliedChanges, WaitingChange{result.Change, result.ClusterChange})
+			appliedChanges = append(appliedChanges, WaitingChange{result.Change, result.ClusterChange, time.Time{}})
```

Comment:
we should just initiate this time field with time.Now() here instead of doing it later.

Reply:
Yeah this will be. better. Done